tests/cypress/latest/support/commands.ts

```diff
@@ -250,17 +251,22 @@ Cypress.Commands.add('createMachReg', ({
     cy.getBySel('registration-url')
       .contains(/https:\/\/.*elemental\/registration/);
 
-    // Try to download the registration file and check it
-    cy.getBySel('download-btn')
-      .click();
-    cy.verifyDownload(machRegName + '_registrationURL.yaml');
-    cy.contains('Saving')
-      .should('not.exist');
-
     // Test ISO building feature
-    if (checkIsoBuilding) {
+    if (checkIsoBuilding && utils.isK8sVersion('rke2')) {
+      // We use dev ISO for main scenario
       cy.getBySel('select-os-version-build-iso')
-        .contains('Elemental Teal');
+      .click();
+      if (utils.isOperatorVersion('stable')) {
+        cy.contains('Elemental Teal ISO x86_64 v1.1.5')
```

Comment:
yes, the `stable` version (1.1.5) is hardcoded, maybe I can delete it and it will choose the first entry but how to be sure it will be always `stable`.  Maybe we can use the `OPERATOR_VERSION` as a variable. I think I will improve it in a next PR though.